cloudnet-modules/cloudnet-report/src/main/java/de/dytanic/cloudnet/ext/report/command/CommandReport.java

```diff
@@ -6,7 +6,7 @@
 import de.dytanic.cloudnet.CloudNet;
 import de.dytanic.cloudnet.cluster.IClusterNodeServer;
 import de.dytanic.cloudnet.command.Command;
-import de.dytanic.cloudnet.command.CommandInfo;
+import de.dytanic.cloudnet.common.command.CommandInfo;
```

Comment:
huh. why this?

Reply:
The CommandInfo wouldn't be accessable from the driver

Reply:
huh, but isn't the common stuff in the driver?

Reply:
Yes, but the CommandInfo was in the node before and therefore not accessable from the driver

Reply:
ah